Orange/widgets/data/owfeaturestatistics.py

```diff
@@ -515,60 +505,45 @@ def data(self, index, role):
                         scene.addItem(histogram)
                         self.__distributions_cache[row] = scene
                     return self.__distributions_cache[row]
-        elif column == self.Columns.CENTER:
-            if role == Qt.DisplayRole:
-                if isinstance(attribute, DiscreteVariable):
-                    output = self._center[row]
-                    if not np.isnan(output):
-                        output = attribute.str_val(self._center[row])
-                elif isinstance(attribute, TimeVariable):
-                    output = attribute.str_val(self._center[row])
-                else:
-                    output = self._center[row]
-        elif column == self.Columns.DISPERSION:
-            if role == Qt.DisplayRole:
+            elif column == self.Columns.CENTER:
+                return render_value(self._center[row])
+            elif column == self.Columns.DISPERSION:
                 if isinstance(attribute, TimeVariable):
-                    output = format_time_diff(self._min[row], self._max[row])
-                else:
-                    output = self._dispersion[row]
-        elif column == self.Columns.MIN:
-            if role == Qt.DisplayRole:
-                if isinstance(attribute, DiscreteVariable):
-                    if attribute.ordered:
-                        output = attribute.str_val(self._min[row])
-                elif isinstance(attribute, TimeVariable):
-                    output = attribute.str_val(self._min[row])
+                    return format_time_diff(self._min[row], self._max[row])
+                elif isinstance(attribute, DiscreteVariable):
+                    return "%g" % self._dispersion[row]
```

Comment:
This produces quite a bit more decimals then the previous rule of (always) 2 decimals.
And it usually looks a bit too much.
E.g. dispersion for the iris class (categ. var.) was previously reported as `1.10` and now `1.09861`

Maybe `"%.2g"` or `"%.3g"` would be better?
E.g. for the above iris example .2g produces just `1.1`
1/3 results in `0.33`, 1/3000 = `0.00033`, and 1/30000 = `3.3e-05`

I don't think precision>3 is needed for categ. vars in most cases. But I am not sure about 2 for cases like the `1.1` above (while `0.33` seems more acceptable).

Thoughts?

Reply:
You're right. I changed it to `"%.3g"`.